subprojects/gradle/build-properties/src/main/kotlin/com/avito/android/info/BuildPropertiesExtension.kt

```diff
@@ -0,0 +1,11 @@
+package com.avito.android.info
+
+open class BuildPropertiesExtension {
+
+    internal val properties = mutableMapOf<String, String>()
+
+    fun buildProperty(name: String, value: String) {
```

Comment:
This will be an input for a `WriteProperties` task. It supports deferrable properties.
We can do it later without breaking backward compatibility for clients.